src/structures/TagBuilder.js

```diff
@@ -257,13 +257,13 @@ TagBuilder.util = {
     error(subtag, context, message) {
         return bbEngine.addError(subtag, context, message);
     },
-    parseChannel(context, channelId) {
+    async parseChannel(context, channelId, args = { quiet: true, supppress: false }) {
         let channel = context.channel;
         if (channel.id !== channelId) {
             if (!/([0-9]{17,23})/.test(channelId))
```

Comment:
This line checks if the channelId is either a mention or an ID, if not returns `Channel not found`, while context.getChannel does provide support for channelnames. Shouldn't it be changed to check if it's an ID, and if so set `channelId` to the matched string instead of the error?

Reply:
To summarize:
- `bu.getChannel` takes either a name or an id and returns the GuildChannel object
- `bu.parseChannel` takes a text and checks for an id (resolved from a mention within the text) and returns the GuildChannel object
- `Builder.util.parseChannel` takes a name or an id (or a mention before changing it to an id). If it can resolve the channel, it returns the GuildChannel object. If it can't, it feeds `context.getChannel` with it.
- `context.getChannel` takes a name or an id. If it can resolve the GuildChannel object, it does. If it can't, it feeds `bu.getChannel` with the data

Reply:
_Check my last commit for the changes :wink:_